packages/livekit-server-sdk/src/SipClient.ts

```diff
@@ -186,6 +187,17 @@ export interface SipTrunkUpdateOptions {
   metadata?: string;
 }
 
+export interface SipOutboundTrunkUpdateOptions {
+  numbers?: ListUpdate;
+  allowedAddresses?: ListUpdate;
+  allowedNumbers?: ListUpdate;
+  authUsername?: string;
+  authPassword?: string;
+  destinationCountry?: string;
+  name?: string;
+  metadata?: string;
```

Comment:
what is metadata used for for outbound?

Reply:
It is defined on the Outbound trunk, but I can't find any usage of it. Intent may have been to set a default value for the ParticipantMetadata, but I'm not sure.